dev/build.changeDetector/test-resources/3581.diff

```diff
@@ -0,0 +1,181 @@
+dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.mongodb-2.0-ssl.feature
```

Comment:
Is this diff file supposed to be checked in?

Reply:
These were used for unit tests I believe